etc/adminhtml/system.xml

```diff
@@ -134,6 +134,12 @@
                     <source_model>Bitpay\BPCheckout\Model\Config\Source\CheckoutSuccess</source_model>
                 </field>
 
+                <field id="bitpay_invoice_close_handling" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Invoice Close Handling</label>
+                    <comment><![CDATA[If this is set to <b>Delete Order</b>, then the order will be deleted when the user closes the BitPay Invoice without paying the invoice.<br><br>If this is set to <b>Keep Order</b>, then the order will not be deleted.]]></comment>
```

Comment:
@p-maguire, @swlodarski-sumoheavy one thing that can happen is this:

1. Invoice is created
2. User takes a picture of the QR code on their phone
3. User closes the BitPay Invoice
4. Order is deleted
5. User pays the invoice

This can result in a paid invoice for a missing order. It's legacy behavior and this PR helps maintain that as default, but -- I think we might want to just change this phrasing a little bit?

> ... then the order will be deleted when the user closes the BitPay Invoice before paying the invoice.

Just `before` instead of `without`. It's small but I think it makes a difference?